modules/help.nf

```diff
@@ -12,27 +12,53 @@ def helpStatement() {
 
     Profiles:
         -profile [conda,nml]            Configuration profile to use. Can use both if separated by a comma
+            conda                         Utilize conda to control tool and dependency installation (uses mamba for environment install)
+            nml                           NML specific profile to take advantage of NML cluster resources
 
     Pick a Pipeline:
         --illumina                      Run Illumina BWA_MEM based competitive host removal pipeline on paired fastq files
         --nanopore                      Set Nanopore data as input, MUST pick a sub pipeline
-            --nanostripper                Run Nanopore host removal based on input fast5 files and nanostripper (NEEDS GUPPY INSTALLED)
             --minimap2                    Run Nanopore host removal based on input fastq files and minimap2
+            --nanostripper                Run Nanopore host removal based on input fast5 files, nanostripper, and guppy (NOT CURRENTLY MAINTAINED)
 
     Illumina Pipeline Arguments:
         Mandatory:
             --directory [path]          Path to directory containing paired fastq files to dehost
-        
+
         Optional:
-            --composite_bwa_index [path]    Path to directory containing BWA indexed composite genome
+            --composite_bwa_index [path]    Path to directory containing BWA indexed composite genome (Note: still need human reference fasta)
+            --keep_ref_id [str]             String name of reference contig ID to keep during host removal (Default: 'MN908947.3')
             --keep_min_map_quality [i]      Integer minimum mapping quality of COVID-19 reads to keep (Default: 60)
             --remove_min_map_quality [i]    Integer minimum mapping quality of HUMAN reads to remove (Default: 0)
 
         Example Command:
             nextflow run phac-nml/ncov-dehoster --human_ref ./hg38.fa -profile conda --illumina --directory ./paired_fastqs
 
+                                        ------------------------------------------
+
+    Minimap2 Pipeline Arguments:
+        Mandatory:
+            --fastq_directory [path]            Path to directory containing either barcoded directories OR individual fastq files
+            --run_name [str]                    Seperate results based on a run name input
+
+        Optional:
+            --fast5_directory [path]            Path to fast5 directories associated with the data
+            --composite_minimap2_index [file]   Path to composite minimap2 .mmi file to speed up analysis
+            --keep_ref_id [str]                 String name of reference contig ID to keep during host removal (Default: 'MN908947.3')
+            --min_length [i]                    Minimum length of fastq reads to keep (Default: 400)
```

Comment:
Was this min_length also set to 350 as a default?

Reply:
Yep, thanks for catching my miss on that. I've fixed it now